mkdocs.yml

```diff
@@ -19,7 +19,9 @@ theme:
 # in which pages show up.
 nav:
   - Home: index.md
-  - Development Guide: devguide.md
+  - Development:
+      - Dev Guide: development/devguide.md
```

Comment:
This should impact the URL for the devguide, right? In which case we will need to update the url mentioned in the `mkdocs/index.md` file as well.